taskvine/src/worker/vine_worker.c

```diff
@@ -490,6 +490,27 @@ static void report_worker_ready(struct link *manager)
 	send_keepalive(manager, 1);
 }
 
+int start_function(struct vine_process *p)
```

Comment:
I don't think it's necessary to search the process table here b/c p->library_process was already set to a stumble library via `process_ready_to_run_now`.

Reply:
Is it possible for this function to fail?  If so, there should be some path by which it returns zero, and then we handle the failure.  If not, just make it return void.

Reply:
This makes sense. Fixed!